cli/smartcontract/smart_contract.go

```diff
@@ -362,13 +392,15 @@ func invokeFunction(ctx *cli.Context) error {
 
 func invokeInternal(ctx *cli.Context, signAndPush bool) error {
 	var (
-		err         error
-		gas         util.Fixed8
-		operation   string
-		params      = make([]smartcontract.Parameter, 0)
-		paramsStart = 1
-		resp        *result.Invoke
-		acc         *wallet.Account
+		err            error
+		gas            util.Fixed8
+		operation      string
+		params         = make([]smartcontract.Parameter, 0)
+		paramsStart    = 1
+		cosigners      = make([]transaction.Cosigner, 0)
```

Comment:
Please make it `nil` by default. You can always append to `nil`, but it will make a better RPC request for us when there are no cosigners specified.